playbooks/robusta_playbooks/job_restart_on_oomkilled.py

```diff
@@ -0,0 +1,196 @@
+from robusta.api import *
```

Comment:
@Avi-Robusta already worked on getting oom killed container, maybe you can reuse his code.
here is a snippet.
```
find_most_recent_oom_killed_container(pod, container_statuses=container_statuses, only_current_state=True)
PodContainer.get_requests(container)
PodContainer.get_limits(container)
```